types/openpgp/index.d.ts

```diff
@@ -4968,6 +4989,8 @@ export interface DecryptResult {
  *          ]
  *          }
  */
+export function decrypt(options: DecryptOptions & { format: "utf8" }): Promise<DecryptOptions & { data: string | ReadableStream<String> | NodeStream }>
+export function decrypt(options: DecryptOptions & { format: "binary" }): Promise<DecryptOptions & { data: Uint8Array | ReadableStream<Uint8Array> | NodeStream }>
```

Comment:
The above Promises should be returning a DecryptResult, not DecryptOptions.

Reply:
Thx, fixed in PR <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36890> :)